apps/cross-components/.storybook/main.js

```diff
@@ -0,0 +1,8 @@
+const rollupStorybook = require('../rollup.storybook');
```

Comment:
I think we should move this to `packages/`. Its not an app. I'm not sure `cross-components` is the right name either.